### ISSUE TITLE
[A11y] Work Show Page-All - Tables

### ISSUE THREAD

**Issue by jechols** [OregonDigital/OD2#2018]
### Operating System

Windows

### Browser

Edge

### Affected page

Work Show

### Admin user?

Yes

### Issue Description

https://staging.oregondigital.org/concern/generics/df727084j

FYI: these are not hard failures per WCAG as I understand it.

The "WORKS AND FILES" table needs some work:

- The "Title" cells should all be row-level headers (`<th scope="row">`)
- "WORKS AND FILES" is essentially captioning the table for sighted users but not for screen-reader users - a table caption would help here, but `aria-labelledby` would also suffice.
- In "WORKS AND FILES", the thumbnail's accessible name/names is/are always "file details". This would be fine if there were a row header because the row header would provide context. It would be describing exactly what it links you to **if** it's given context. One could argue to just add invisible text to it, but then you lose out on the value of having row-level context (or else you duplicate it), and you have magic hidden text, which is always best to avoid when there's a way to associate visible text instead.

The "User Activity" table might make more sense to just not be a table, or else be a layout table. The headers add unnecessary verbosity and a navigation burden considering the items could effectively be read linearly and make sense.

### Suggested Fix

- Set "title" cells to a row-level header in all tables that have a title column
- Give the "WORKS AND FILES" table a caption / label
- Consider whether the user activity table could be presented in a simpler form. If a table is best/easiest, give it a caption / label

### Can anybody QA or does this need an expert?

Needs expert

### QA Steps

Verification of this kind of thing is most easily done with a screen reader. Otherwise:

- Look over the HTML and verify that the table's "Title" cells are all set as row-level headers
- Make sure tables have labels or captions
- Verify that existing table semantics weren't compromised / removed (by accident)
